modules/affile/collection-comparator.c

```diff
@@ -112,7 +116,7 @@ collection_comparator_add_value(CollectionComparator *self, const gint64 key[2],
       entry->value = g_strdup(value);
       entry->flag = IN_NEW;
       self->original_list = g_list_append(self->original_list, entry);
-      g_hash_table_insert(self->original_map, (void*) key, entry);
+      g_hash_table_insert(self->original_map, entry, entry);
```

Comment:
For clarity's sake please use different names for the parameters.

Reply:
the parameters cannot be different, the key and the data is the same